client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -156,9 +156,9 @@ function CalypsoWrappedMarketplaceDomainUpsell(): JSX.Element {
 						previousPath
 							? page( previousPath )
 							: page(
-									`/plugins/marketplace/product/details/wordpress-seo${
+									`/marketplace/product/details/wordpress-seo${
 										selectedSite?.slug ? `/${ selectedSite.slug }` : ''
-									}`
+									}?flags=marketplace-yoast`
```

Comment:
Isn't that mostly used for dynamic data?

Reply:
It can be used for both static and dynamic data; the idea is that we abstract the logic for how to append a query arg to the function, rather than making a decision for whether it to be a question mark or an ampersand ourselves.

Reply:
Ok will have a look at it in the future 👍